layers/bike.yaml

```diff
@@ -0,0 +1,1220 @@
+    cycle_track: { any: [ { kind_detail: true, cycleway: [track, opposite_track] }, cycleway_left: [track, opposite_track], cycleway_right: [track, opposite_track], kind_detail: [cycleway, path, pedestrian, footway, steps] ] }
+        # any:
+        #     - { kind_detail: true, cycleway: [track, opposite_track] }
+        #     - cycleway_left: [track, opposite_track]
+        #     - cycleway_right: [track, opposite_track]
+        #     - kind_detail: [cycleway, path, pedestrian, footway, steps]
+
+    draw_block_left_or_right_or_implied: { filter: global.left_or_right_or_implied, draw: { casing_left: { color: white }, casing_right: { visible: true, color: global.left_right_lane }}, left_is_special: { filter: global.left_is_special, draw: { casing_left: { visible: true, color: global.left_right_lane }, casing_right: { visible: false, color: white } } } }
+        # left_or_right_or_implied:
+        #     filter: global.left_or_right_or_implied
+        #     draw:
+        #         # lines:
+        #         #     outline:
+        #         #         color: [[12, [0.900,0.900,0.900]], [13, red]]
+        #         casing_left:
+        #             #order: 1000 # function() { return feature.sort_rank + 1; }
+        #             #visible: true
+        #             color: white
+        #         casing_right:
+        #             #order: 1000 # function() { return feature.sort_rank + 1; }
+        #             visible: true
+        #             color: blue
+        #     left_is_special:
+        #         filter: global.left_is_special
+        #         draw:
+        #             casing_left:
+        #                 #order: 1000 # function() { return feature.sort_rank + 1; }
+        #                 visible: true
+        #                 color: blue
+        #             casing_right:
+        #                 #order: 1000 # function() { return feature.sort_rank + 1; }
+        #                 visible: false
+        #                 color: white
+    draw_block_shared_yellow_lane: { filter: global.shared_lane, draw: { lines: { color: global.shared } } }
+        # shared_lane:
+        #     filter: global.shared_lane
+        #     draw:
+        #         lines:
+        #             color: yellow
+
+
+    # COLORS
+    bike: [0.000,0.549,0.929] # blue is route #[0.137,0.584,0.812]
+    bike_path: [0.000,0.565,0.345]
+    bike_separate: [0.110,0.686,0.435] #green is safest
+    left_right_lane: [0.414,0.940,0.607]
+    # left_right_lane: [0.122,0.894,0.400]
+    cycle: [0.000,0.680,0.385]
+    cycle_path: [0.000,0.565,0.345]
+    shared: [1.000,1.000,0.271] #not so safe, biking with cars, no dedicated lane
+
+
+layers:
+    roads:
+        arrows:
+            # oneway arrows and shields are distinct groups!
+            filter: { oneway: yes, shield_text: false, $zoom: { min: 17 } }
+            draw:
+                oneway:
+                    sprite: arrow
```

Comment:
can we set another property here: `flat: true`? Refer: https://mapzen.com/documentation/tangram/draw/#flat

Sorry I know this is WIP, if I am commenting too early here.
P.S. We will need to have this set everywhere we use something like an arrow.